src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/domain/makerecalldecisions/recommendation/BookRecallToPpud.kt

```diff
@@ -14,6 +14,7 @@ data class BookRecallToPpud(
   val indexOffence: String? = null,
   val indexOffenceComment: String? = null,
   val ppudSentenceId: String? = null,
+  val ppudSentenceData: IndeterminateSentenceData? = null,
```

Comment:
I think the field name should also include the word 'indeterminate', to reduce the chances of taking from it for determinate cases (and maybe we eventually define a similar field for determinate cases?).

Reply:
I see what you mean but does that disassociate it from the ppudSentenceId which this data relates to? I wander if there is something between the two? 🤔